enhancements/update/automatic-updates.md

```diff
@@ -0,0 +1,166 @@
+Cluster update suggestions are distributed via Cincinnati][cincinnati-openshift], so clusters only receive update recommendations that are appropriate for their cluster.
+While manually approving updates at a per-cluster level is possible, it should not be required.
+This enhancement adds a new `automaticUpdates` property to [the ClusterVersion `spec`][api-spec] so cluster adminstrators can opt in to or out of automatic updates.
+
+## Motivation
+
+Manually approving updates is tedious, doesn't scale well for administrators managing many clusters, and delays the application of potential security fixes.
+Administrators may wish to enable automatic updates for any of those or other reasons.
+Currently, administrators must implement custom polling logic to check for and apply any available updates.
+This enhancement would provide those administrators with a convenient property instead.
+
+### Goals
+
+* Make it easy to opt in to and out of automatic updates.
+
+### Non-Goals
+
+* Support calendar gating or other logic about when updates may be attempted.
+* Choose whether the installer defaults to enabling automatic updates or not for new clusters.
+
+## Proposal
+
+As proposed in [this API pull request][api-pull-request], to add a new property:
+
+```go
+// automaticUpdates enables automatic updates.
+// +optional
+AutomaticUpdates bool `json:"automaticUpdates,omitempty"`
+```
+
+to [`ClusterVersionSpec`][api-spec] to record the administrator's preference.
+
+### User Stories
+
+#### Canary Clusters
+
+Alice has a canary clusters she wants to update as soon as an updated release is available in her configured channel.
+Before this enhancement she would need to construct a poller to run `oc adm upgrade --to-latest` or similar.
+With this enhancement, she can set `automaticUpdates` and does not need to write an external poller.
+
+### Risks and Mitigations
+
+Cluster-managed updates are a relatively new thing, and while we have dozens of OpenShift 4 releases out so far, we have had a few updates result in stuck clusters (like [this][machine-config-operator-drain-wedge].
+Allowing automatic updates would make it more likely that upgrade failures happen when there is no admistrator actively watching to notice and recover from the failure.
+This is mitigated by:
+
+* Alerting.
+    The cluster should fire alerts (FIXME: which ones?) when an upgrade gets into trouble.
+    Adminstrators can configure the cluster to push those alerts out to the on-call admistrator to recover the cluster.
+* Stability testing.
+    We are continually refining our CI suite and processing Telemetry from live clusters in order to assess the stability of each upgrade.
+    We will not place upgrades in production channels unless they have proven themselves stable in earlier testing, and we will remove upgrades from production channels if a live cluster trips over a corner case that we do not yet cover in pretesting.
```

Comment:
s/remove/stop pushing/

Reply:
> s/remove/stop pushing/

Expanded "we will remove upgrades from production channels" to "we will remove update recommendations from production channels".  We're not really pushing anything, we're just serving graphs to clients like the CVO.  Does the new wording look good to you, or do you have an alternative idea?